src/main/java/com/adyen/util/HMACValidator.java

```diff
@@ -65,7 +65,7 @@ public String calculateHMAC(String data, String key) throws java.security.Signat
     }
 
     // To calculate the HMAC SHA-256
-    public String calculateHMAC(NotificationRequestItem notificationRequestItem, String key) throws SignatureException {
```

Comment:
possible NPE on line#57

Reply:
The reference to `data` on line#57 has a handler for `Exception` around it, so I think it should be fine.